python/rikai/spark/types/video.py

```diff
@@ -93,3 +94,41 @@ def deserialize(self, datum) -> "YouTubeVideo":
 
     def simpleString(self) -> str:
         return "YouTubeVideoType"
+
+
+class SegmentType(UserDefinedType):
+    """Defines the Spark UDT for a video segment"""
+
+    def __init__(self):
+        super().__init__()
+
+    def __repr__(self) -> str:
+        return "SegmentType"
+
+    @classmethod
+    def sqlType(cls) -> StructType:
+        return StructType(
+            fields=[
+                StructField("start_fno", IntegerType(), nullable=False),
+                StructField("end_fno", IntegerType(), nullable=False),
+            ]
+        )
+
+    @classmethod
+    def module(cls) -> str:
+        return "rikai.spark.types"
```

Comment:
It is not a big deal, but I'd lean to use the strictest module as possible (i.e. `rikai.spark.types.video`), i am worried that we will soon get into cycle-import hell.

Reply:
that's fair. I'll change it.